layers/+chat/circe/packages.el

```diff
@@ -0,0 +1,142 @@
+    (setq circe-prompt-string (concat (propertize ">>>" 'face 'circe-prompt-face) " "))
+    (setq circe-highlight-nick-type 'all)
+
+    ;; Add circe-color-nicks
+    (enable-circe-color-nicks)
+    (setq circe-color-nicks-everywhere 't)
+
+    ;; Show channel name in prompt.
+    (add-hook 'circe-chat-mode-hook
+              (lambda ()
+                (-let* (
+                        ;; Get buffername
+                        (s (buffer-name))
+                        ((_ gitter-fmt) (s-match (rx (+ nonl) "/" (group (+ nonl)) eos) s))
+                        ((_ standard-fmt) (s-match (rx "#" (group (+ nonl)) eos) s))
+                        ;; Buffername to channel name
+                        (chan (or gitter-fmt standard-fmt))
+                        ;; Format prompt
+                        (prompt (concat "#" (propertize chan 'face 'circe-prompt-face) " > ")))
+                  (lui-set-prompt prompt))))
+
+    ;; Timestamps in margins.
+    (setq lui-time-stamp-position 'right-margin)
+    (setq lui-time-stamp-format "%H:%M")
+    (setq lui-fill-type nil)
+    (add-hook 'lui-mode-hook (lambda ()
+                               (setq fringes-outside-margins t)
+                               (setq right-margin-width 5)
+                               (setq word-wrap t)
+                               (setq wrap-prefix "    ")))
+
+    (add-hook 'circe-channel-mode-hook #'enable-lui-autopaste)
+
+    ;; Keybindings
+    (spacemacs/set-leader-keys "aic" 'circe)
+
+    (dolist (mode circe-modes)
+      (spacemacs/declare-prefix-for-mode mode "mc" "command")
+      (spacemacs/set-leader-keys-for-major-mode mode
+        "r" 'circe-reconnect
+        "R" 'circe-reconnect-all
+        "v" 'circe-version
+        "c n" 'circe-command-NAMES
+        "c q" 'circe-command-QUERY
+        "c h" 'circe-command-HELP
+        "c t" 'circe-command-CHTOPIC
+        "c s" 'circe-command-SAY
+        ))))
+
+(defun circe/post-init-helm ()
+  (with-eval-after-load 'helm
+    (if (not circe-helm-for-username-completion)
+        (dolist (mode circe-modes)
+          (add-to-list 'helm-mode-no-completion-in-region-in-modes mode)))))
+
+(defun circe/init-helm-circe ()
+  (use-package helm-circe
+    :commands (helm-circe
+               helm-circe-servers
+               helm-circe-queries
+               helm-circe-by-server
+               helm-circe-new-activity)
+    :after circe
+    :config
+    ;; Helm keybindings
+    (dolist (mode circe-modes)
+      (spacemacs/declare-prefix-for-mode mode "mh" "helm")
+      (spacemacs/set-leader-keys-for-major-mode mode
+        "h h" 'helm-circe
+        "h s" 'helm-circe-servers
+        "h q" 'helm-circe-queries
+        "h b" 'helm-circe-by-server
+        "h n" 'helm-circe-new-activity
+        ))))
+
+(defun circe/init-circe-notifications ()
+  (use-package circe-notifications
+    :commands enable-circe-notifications
```

Comment:
The `:commands` should be unnecessary because the circe-notifications package specifies the autoload for `enable-circe-notifications`.